test/error/old_implicit_args.cpp

```diff
@@ -10,7 +10,7 @@ int main(int argc, char **argv) {
     one_arg(x) = x * 2;   // One argument
 
     Func bad_call;
-    bad_call() = one_arg; // Treated as Expr, results in implicit arg, but RHS doesn't have a _.
+    bad_call() = one_arg(_); // Has an implicit arg, but RHS doesn't have a _.
```

Comment:
Oh, I misread RHS as LHS. I'll just delete the test. I agree that it's no longer needed